lib/vsc/administration/slurm/sync.py

```diff
@@ -380,16 +380,16 @@ def slurm_user_accounts(vo_members, active_accounts, slurm_user_info, clusters,
             # these are users not yet in the Slurm DB for this cluster
             new_users |= {
                 (user, vo.vsc_id, vo.institute['name'])
-                for user in (members & active_accounts) - cluster_users
+                for user in members - cluster_users
             }
 
             # these are the current Slurm users per Account, i.e., the VO currently being processed
             slurm_acct_users = {user for (user, acct) in cluster_users_acct if acct == vo_id}
 
             # these are the users that should no longer be in this account, but should not be removed
-            # we need to look up their new VO
+            # we need to look up their new VO. We remove the users which are nog longer active too.
```

Comment:
maybe clarify that inactive users are not considered "changed", but will be completly removed